addons/website/static/src/scss/website.scss

```diff
@@ -1031,6 +1031,18 @@ table.table_desc tr td {
     }
 }
 
+// Navbar Dropdown Menu
+header > .navbar > * .dropdown-menu {
+    max-height: 60vh;
+    overflow-y: auto;
+}
+
+header.o_affix_enabled > .navbar > * .dropdown-menu {
+    @include media-breakpoint-down(md) {
+        max-height: 100%;
+    }
+}
```

Comment:
I was pinged on a forward-port PR for this but I don't understand the original fix. Can you please explain:
- Why using `*` in those scss selector?
- How is the issue related to `o_affix_enabled`?

Reply:
> I think he meant you could just have:
>
>     header > .navbar .dropdown-menu
>

@nle-odoo True. Sorry. I did `header > .navbar > .dropdown-menu` at first and that's what mislead me...
@qsm-odoo For the mobile scrolling, I think NLE explained the problem better than I did. :slightly_smiling_face:

Reply:
ping @qsm-odoo